tools/scripts/build_projects.py

```diff
@@ -54,7 +54,11 @@ def log_success(msg):
 log_file = DEFAULT_LOG_FILE
 
 def run_cmd(cmd):
-	log(cmd)
+	tmp = cmd.split(' ')
```

Comment:
This commit "tools:scripts:build_projects.py: Improve log" could be merged with the previous commit that adds logging to have logging "improved" from the first shot :)

Reply:
They are to separated things. One moves logs to files and this one just make pretty the normal log